### PR TITLE
Add Monmouth County, NJ

### LINKED ISSUE #234: Monmouth County, NJ
http://co.monmouth.nj.us/page.aspx?ID=3866

### PR DESCRIPTION
**Pull request by iandees**
Fixes #234.

### CODE REVIEW

File: sources/us/nj/monmouth.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "coverage": {
+        "US Census": {
+            "geoid": "32005",
+            "name": "Douglas County",
```
Comment (trescube) [inline sources/us/nj/monmouth.json]: Old file?

Comment (iandees) [inline sources/us/nj/monmouth.json]: Yep, copypasta. Thanks for catching.

**Comment by iandees** [comment]
Does anyone have a second to see if the regex I used here works more often than not?

**Comment by migurski** [comment]
This might be a good reason to introduce the U.S. Addresses regexp / function we talked about. @trescube I think you said you’d be interested to work on that? I can help.

**Comment by trescube** [comment]
It's at the top of my short to-do list.

**Comment by migurski** [comment]
Dots look good though:

Monmouth

**Comment by iandees** [comment]
I spot-checked a couple dozen points and the regex seems to have worked, so I'm going to merge this in.

**Comment by ingalls** [comment]
🎉 Thanks folks!

**Pull request merged**